contracts/committee/Committee.sol

```diff
@@ -594,6 +618,27 @@ contract Committee is ConsensusModule {
         return positionCounts[proposal] >= quorum;
     }
 
+    /**
+     * @dev Function requires:
+     *          - only consensus can call
+     *          - proposal was accepted by committee
+     *          - consensus has approved the committee to transfer a reward
+     *            from its address to this contract address.
+     */
+    function rewardCommittee(uint256 _reward)
```

Comment:
so this transfer should happen in consensus (when minted), and it suffices for `committee` to have a function where members can pull funds from `Committee` to their reward balance on `Reputation`, if `committeeDecision` is reached.

this function can be rewritten to store the total reward, and simply check whether the mOST balance is available to the Committee to divide equally over the members who contributed their revealed vote to the committee decision (after reveqal period ends)